webapp/src/components/jira_field.jsx

```diff
@@ -175,7 +175,7 @@ export default class JiraField extends React.Component {
                 }
 
                 const onChange = (id, val) => {
-                    const newValue = val.map((v) => ({id: v}));
+                    const newValue = val ? val.map((v) => ({id: v})) : [];
```

Comment:
```suggestion
                    const newValue = val?.map((v) => ({id: v})) ?? [];
```

Reply:
optional chaining is not present in the JIRA plugin and there is an open ticket for that as well. So I think we have to use ternary operator here

Reply:
okay then we can keep it this way